webpack.config.js

```diff
@@ -178,6 +178,8 @@ function* webpackConfig( env, argv ) {
 				'./assets/js/googlesitekit-modules-search-console.js',
 			'googlesitekit-modules-subscribe-with-google':
 				'./assets/js/googlesitekit-modules-subscribe-with-google.js',
+			'googlesitekit-subscribe-with-google-bulk-edit':
+				'./assets/js/modules/subscribe-with-google/bulk-edit.js',
```

Comment:
Instead of doing that, please create an `assets/js/googlesitekit-subscribe-with-google-bulk-edit.js` file as entry point - all our entry points should be located in `assets/js`.